src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -124,19 +133,27 @@ def stop_process(proc):
         return procrtn
 
     def get_env(self, cartobj):
+        self.stdout.info("Entering get_env")
         """ return basic env setting in yaml """
         env_CCSA = cartobj.params.get("env", "/run/env_CRT_CTX_SHARE_ADDR/*/")
         test_name = cartobj.params.get("name", "/run/tests/*/")
         host_cfg = cartobj.params.get("config", "/run/hosts/*/")
 
+        self.stdout.info("\ncart_utils: line 146, ENV : %s\n" % os.environ)
```

Comment:
(pylint-logging-not-lazy) Specify string format arguments as logging function parameters